_includes/templates/office-contact.html

```diff
@@ -0,0 +1,22 @@
+{% set contact = get_document('contact', office.related_contact) if office.related_contact %}
+{% if contact %}
+	{% import "contact-macro.html" as contact_macro %}
+    <section class="office_contact
+    				block
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/b93d678e79f5a3a8beed84f222446e373de6949d